src/components/custom/_signup-ctrader-form.tsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react'
```

Comment:
in case when this component using atomic approach, let's move it to `features` folder

Reply:
Nikita I've created many components and all are using new atomic approach. Should I move all of them into features folder?

Reply:
Yes Mohammad, you have 🙏